site/content/blog/2025-06-01-zap-updates-may-2025/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "ZAP Updates - May 2025"
+summary: >
+  May 2025 updates and ongoing feature development statuses.
+images:
+- https://www.zaproxy.org/blog/2025-06-01-zap-updates-may-2025/images/zapbot-monthly-updates.png
+type: post
+tags:
+- blog
+- update
+date: "2025-06-01"
+authors:
+- thorin
+---
+
+## Highlights
+TODO
+
+## Ongoing Work
+
+### LLM Add-on Pull Request
+This experimental feature aims to leverage Large Language Models (LLMs)  within ZAP in a meaningful way.
+:tada: Since last month [Akshath](https://www.zaproxy.org/docs/team/ricekot/) has added some functionality to the PR to facilitate testing and use with local LLMs (for example via Ollama) :tada:
+
+### Continued Enhancements to Authentication
+Work is steadily progressing to make authentication even more reliable and adaptable.
+We continue to find new challenges in how the world performs authentication, but are evolving continually.
+
+## GitHub Pulse
+Here are some statistics for the two main ZAP repositories:
+
+[zaproxy](https://github.com/zaproxy/zaproxy/pulse/monthly)
+Excluding merges, 6 authors have pushed 8 commits to main and 8 commits to all branches. On main, 16 files have changed and there have been 647 additions and 80 deletions. .
+
+[zap-extensions](https://github.com/zaproxy/zap-extensions/pulse/monthly)
+Excluding merges, 5 authors have pushed 66 commits to main and 66 commits to all branches. On main, 257 files have changed and there have been 3,302 additions and 572 deletions.
+
+A total of [92 human PRs were merged](https://github.com/search?q=org%3Azaproxy+type%3Apr+-author%3Azapbot+-author%3Aapp%2Fdependabot+sort%3Aupdated-asc+closed%3A2025-05+is%3Amerged&type=pullrequests) on the ZAP repos.
+
+## Released Add-ons - Full Changelog
+In May 2025, we released updated versions of 5 add-ons:
+
+##### Dev Add-on
+**v0.10.0**  
+Added
+- Basic CSRF test app.
+- Page with input elements that appear after a delay and off the displayed screen.
+- Auth app which uses multiple (faked) domains.
+- An auth example where there's a div that may obscure the login fields.
+
+##### Linux WebDrivers
+**v141**  
+Changed
+- Update ChromeDriver to 137.0.7151.55.
+
+<details>
+<summary>v138 .. v140</summary>
+
+**v140**  
+Changed
+- Update ChromeDriver to 136.0.7103.113.
+
+**v139**  
+Changed
+- Update ChromeDriver to 136.0.7103.94.
+
+**v138**  
+Changed
+- Update ChromeDriver to 136.0.7103.92.
+
+</details>
```

Comment:
I'm not sure if we can do this in the draft, but I decided to roll these up since they don't really add anything terribly interesting for the majority of readers.

Reply:
That does not work.